src/pages/contact.tsx

```diff
@@ -72,13 +23,105 @@ const ThankYouMessage = () => (
   </div>
 )
 
-const Contact = () => {
+const ContactForm = () => {
   const [contactInfoSent, setContactInfoSent] = useState<boolean>(false)
+  const [termsOfServiceAccepted, setTermsOfServiceAccepted] = useState<boolean>(false)
 
-  function onInfoSent () {
-    setContactInfoSent(true)
+  const [email, setEmail] = useState('')
+  const [name, setName] = useState('')
+  const [message, setMessage] = useState('')
+  const [nameError, setNameError] = useState('')
+  const [emailError, setEmailError] = useState('')
+  const [termsOfServiceError, setTermsOfServiceError] = useState('')
+
+  function resetErrors () {
+    setNameError('')
+    setEmailError('')
+    setTermsOfServiceError('')
+  }
+
+  function handleSendClick () {
+    resetErrors()
+    if (!validateName(name)) setNameError(websiteCopyStrings.contactFormNameError)
+    else if (!validateEmail(email)) {
+      setEmailError(websiteCopyStrings.contactFormEmailError)
+    } else if (!termsOfServiceAccepted) {
+      setTermsOfServiceError(websiteCopyStrings.contactFormTermsOfServiceError)
+    } else {
+      setContactInfoSent(true)
+      alert(message)
+    }
   }
 
+  return contactInfoSent
+    ? (
+    <ThankYouMessage />
+      )
+    : (
+    <div className="bg-gray-100 flex flex-col items-center md:rounded-3xl pt-8 pb-10 px-3">
+      <h1 className="text-xl font-bold mb-3 ">
+        {websiteCopyStrings.contactFormHeaderPart1}{' '}
+        <span className="text-3xl">{websiteCopyStrings.contactFormHeaderPart2}</span>
+      </h1>
+      <p className="text-gray-600 mb-16 text-lg">{websiteCopyStrings.contactFormMessageCopy}</p>
+      <div className="md:w-8/12">
+        <input
+          className="bg-white w-full rounded-lg mb-3 px-4 py-3"
+          id="name"
+          aria-label="name"
+          type="text"
+          onChange={(e) => setName(e.target.value)}
+          placeholder={websiteCopyStrings.contactFormName}
+        />
+        <div className="text-red-400 mb-8 ml-4 float-left">{nameError}</div>
+        <input
+          className="bg-white w-full rounded-lg  mb-3 px-4 py-3"
+          id="email"
+          type="text"
+          aria-label="email"
+          onChange={(e) => setEmail(e.target.value)}
+          placeholder={websiteCopyStrings.contactFormEmail}
+        />
+        <div className="text-red-400 mb-8 ml-4 float-left">{emailError}</div>
+
+        <textarea
+          className="bg-white w-full rounded-lg mb-8  px-4 py-3"
+          id="message"
+          aria-label="message"
+          onChange={(e) => setMessage(e.target.value)}
+          placeholder={websiteCopyStrings.contactFormMessage}
+          rows={9}
+        />
+        <div className="text-left flex">
+          <div>
+            <input
+              onChange={() => setTermsOfServiceAccepted(!termsOfServiceAccepted)}
+              type="checkbox"
+              className="form-checkbox"
```

Comment:
> 4:40:09 PM: [Jina AI | Jina AI is a Neural Search Company (contact.html)] error 1: This checkboxinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined. (<input type="checkbox" class="form-checkbox">)
4:40:09 PM: [Jina AI | Jina AI is a Neural Search Company (contact.html)] error 1: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate. (<input type="checkbox" class="form-checkbox">)
```suggestion
              className="form-checkbox"
              aria-label="accept terms of service checkbox"
```